### DISCUSSION TITLE
BUG: Pinned discussions has scroll bars

### DISCUSSION THREAD

**Discussion by mmvanheusden** [community/community#2995] [Discussions] [answered]
exactly what the title says, Pinned discussions have scroll bars
[screenshot](https://imgur.com/a/U268wyH)

**Comment by evi-liu** [comment]
Thanks for the report! I'm not able to reproduce. Which browser are you seeing this on?

**Reply by c0b41** [reply]
this only happening windows.

**Comment by mmvanheusden** [comment]
The latest version of Google Chrome

**Comment by becca** [comment, marked as answer]
This should now be fixed

**Reply by mmvanheusden** [reply]
Yes it is, thak you!

**Reply by jonathanfallon** [reply]
Yes it is fixed. Thanks !
